go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -31,6 +31,10 @@ func NewL2SwitchManager() *LogicalSwitchManager {
 	}
 }
 
+func NewL2SwitchManagerForUserDefinedPrimaryNetwork() *LogicalSwitchManager {
+	return NewLogicalSwitchManager()
```

Comment:
Let's be explicit and just do, this way is easier to compare with the `NewL2SwitchManager`.
```golang
return &LogicalSwitchManager{
		allocator:  subnet.NewAllocator(),
		reserveIPs: true,
	}
```

Reply:
that'll be flagged as duplicate code by any tool that checks that.